plugins/module_utils/openssh/backends/common.py

```diff
@@ -75,7 +77,14 @@ def __init__(self, module):
         self.check_mode = self.module.check_mode
 
     def execute(self):
-        self._execute()
+        try:
+            self._execute()
+        except Exception as e:
+            self.module.fail_json(
+                msg="unexpected error occurred",
```

Comment:
I would still include the exception's name into the error message:
```suggestion
                msg="unexpected error occurred: %s" % to_native(e),
```

Reply:
Added back